### PR TITLE
docs: update mdl migration docs

### PR DESCRIPTION
**Pull request by ChildishGiant**

### CODE REVIEW

**Comment by ChildishGiant** [comment]
@googlebot I signed it!

File: docs/migrating-from-mdl.md [hunk 1]
```diff
@@ -319,11 +319,11 @@ The following table summarizes the current situation (TBI = to be investigated):
 | `mdl-shadow` | [`@material/elevation`](../packages/mdc-elevation/README.md) | Similar usage. |
 | `mdl-slider` | [`@material/slider`](../packages/mdc-slider/README.md) | Very different DOM. |
 | `mdl-snackbar` | [`@material/snackbar`](../packages/mdc-snackbar/README.md) | Very different DOM. |
-| `mdl-spinner` | TBI | [#30](https://github.com/material-components/material-components-web/issues/30) |
+| `mdl-spinner` | [`@material/progress-indicator`](../packages/mdc-progress-indicator/README.md) |  |
```
Comment (allan-chen) [inline docs/migrating-from-mdl.md:322]: Please change to `mdc-circular-progress`

**Review by allan-chen — COMMENTED** [review]
Thanks for the PR! Just a nit change

**Review by asyncLiz — APPROVED** [review]
Sorry for the delay on this!

**Pull request merged**